<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singup</class>
 <widget class="QMainWindow" name="singup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1641</width>
    <height>1879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/back47.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>1301</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/back40.jpg);</string>
    </property>
    <property name="title">
     <string>Signup Page</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_en_7">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>70</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>140</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">x</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_en_8">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>440</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>210</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Type </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_en_9">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>280</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass_4">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>530</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>210</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Student</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>360</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>530</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Contact no</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>440</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass_6">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>360</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>210</y>
       <width>270</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Administrator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>660</y>
       <width>250</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

padding : 0 8px;
background: orange;
border -radius:10px;
}</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>140</y>
       <width>351</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/xen.jpeg);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/back41.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>-10</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
padding :0 8px;
background: light red;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>-10</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

padding :0 8px;
background: light red;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>About US</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>-10</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

padding :0 8px;
background: light red;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>Webservices</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
